shaders/fragment.glsl

```diff
@@ -16,10 +17,9 @@ varying vec3 lightDirection, eyeDirection, surfaceNormal;
 varying vec4 vColor;
 
 void main() {
-  if (kill > 0.0 ||
-    any(lessThan(worldCoordinate, clipBounds[0])) || any(greaterThan(worldCoordinate, clipBounds[1]))) {
-    discard;
-  }
+  if (kill > 0.0) discard;
+
+  if (outOfRange(clipBounds[0], clipBounds[1], worldCoordinate)) discard;
```

Comment:
This is definitely more readable... but it does seem to be established practice to minimize `if` statements:

> In particular, if you can remove an if statement from a shader, that will make it run faster.

https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/WebGL_best_practices

So probably better to collapse it back to a single `if`.